.travis.yml

```diff
@@ -6,9 +6,16 @@ before_script:
 
 after_success: ./travis-nightly.sh
 
+sudo: false
+
```

Comment:
They [updated something in Travis](http://docs.travis-ci.com/user/workers/container-based-infrastructure/) and this property should help speed up things